### PR TITLE
Feature/unit tests notify

### PR DESCRIPTION
**Pull request by Lex-23**
created unit test for func notify

### CODE REVIEW

**Comment by Lex-23** [comment]
[Write tests for first part Notification](https://trello.com/c/X1NdNG0s/36-write-tests-for-first-part-notification)

File: cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    ],
+)
+@patch(
+    "utils.notification_handlers.telegram_client.bot.send_message",
+    return_value="success telegram notify",
+)
+def test_telegram_notify_success(send_message, notification_type):
+    salesdasboard = SalesDashboardFactory()
+    offer = OfferFactory(asset=salesdasboard.asset, broker=salesdasboard.broker)
+    data = {
```
Comment (vadim-denisov) [inline cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:27]:
```suggestion
    data_expected = {
```

File: cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            "asset_name": salesdasboard.asset.name,
+        },
+        NotificationType.SALESDASHBOARD_SOON_OVER: {
+            "salesdashboard_id": salesdasboard.id
+        },
+    }
+    account = salesdasboard.broker
+
+    consumer = ConsumerFactory(
+        account=account, type=ConsumerType.TELEGRAM, data={"tg_chat_id": 00000000}
```
Comment (vadim-denisov) [inline cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:40]:
```suggestion
        account=account, type=ConsumerType.TELEGRAM, data={"tg_chat_id": 0}
```

Comment (vadim-denisov) [reply cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:40]: Or it should be string?

Comment (Lex-23) [reply cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:40]: No, int is right

File: cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py (hunk 1 above)
Comment (Lex-23) [reply cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:27]: ok

File: cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py (hunk 2 above)
Comment (Lex-23) [reply cryptostock_api/cryptostock/notification/tests/unit_tests/test_func_notify.py:40]: got it

**Review by vadim-denisov — APPROVED** [review]

**Pull request merged**